cg/meta/transfer/lims.py

```diff
@@ -114,6 +132,38 @@ def transfer_pools(self, status_type: PoolState):
                     else:
                         continue
 
+    def transfer_microbial_samples(self, status_type: MicrobialState):
+        """Transfer information about microbial samples."""
+
+        microbial_samples = self._microbial_samples_functions[status_type]()
+        result = []
+
+        if microbial_samples is None:
+            LOG.info(f"No microbial samples found with {status_type.value}")
+            return
+        else:
+            LOG.info(f"Processing {microbial_samples.count()} microbial samples")
+
+        for microbial_sample_obj in microbial_samples:
+            internal_id = microbial_sample_obj.internal_id
+
+            lims_date = self._date_functions[status_type](microbial_sample_obj.internal_id)
+            statusdb_date = getattr(microbial_sample_obj, f'{status_type.value}_at')
+            if lims_date:
+
+                if statusdb_date and statusdb_date.date() == lims_date:
+                    continue
+
+                LOG.info(f"Found new {status_type.value} date for {microbial_sample_obj.internal_id}: " \
+                         f"{lims_date}, old value: {statusdb_date} ")
```

Comment:
This doesn't need to be `statusdb_date.date()`?

Reply:
statusdb_date is NoneType if no date is found and therefore has no date() method, that's probably why it's used like that in transfer_samples where I blatantly copied this from :)